packages/api/src/external/carequality/process-outbound-patient-discovery-resps.ts

```diff
@@ -0,0 +1,79 @@
+import { Patient } from "@metriport/core/domain/patient";
+import { out } from "@metriport/core/util/log";
+import { OutboundPatientDiscoveryRespResults } from "@metriport/core/external/carequality/ihe-gateway/outbound-result-pooler-direct";
+import { capture } from "@metriport/core/util/notifications";
+import { OutboundPatientDiscoveryResp } from "@metriport/ihe-gateway-sdk";
+import { errorToString } from "@metriport/shared/common/error";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { createOrUpdateCQPatientData } from "./command/cq-patient-data/create-cq-data";
+import { CQLink } from "./cq-patient-data";
+
+dayjs.extend(duration);
+
+const context = "cq.patient.discover";
+
+export async function processOutboundPatientDiscoveryResps({
+  requestId,
+  patientId,
+  cxId,
+  results,
+}: OutboundPatientDiscoveryRespResults): Promise<void> {
+  const baseLogMessage = `CQ PD Processing results - patientId ${patientId}`;
+  const { log } = out(`${baseLogMessage}, requestId: ${requestId}`);
+  const { log: outerLog } = out(baseLogMessage);
+
+  try {
+    if (results.length === 0) {
+      log(`No patient discovery results found.`);
+      return;
+    }
+
+    log(`Starting to handle patient discovery results`);
+    await handlePatientDiscoveryResults(
+      {
+        id: patientId,
+        cxId,
+      },
+      results
+    );
+
+    log(`Completed.`);
+  } catch (error) {
+    const msg = `Error on Patient Discovery`;
```

Comment:
nit: perhaps being explicit on outbound PD would help us identify the Sentry messages quicker